README.md

```diff
@@ -1,33 +1,28 @@
-<p align="center">
-  <h1>
-  St. Jude Cloud Utilities
-  </h1>
-  <a href="https://github.com/stjudecloud/utilities/blob/master/LICENSE.md" target="_blank">
-    <img alt="License: MIT" src="https://img.shields.io/badge/License-MIT-yellow.svg" />
-  </a>
-</p>
+# St. Jude Cloud Utilities
 
-> This repository contains utility scripts used on the St. Jude Cloud project.
+This repository contains an easily installable package for all of the utility scripts created for and used on the St. Jude Cloud team (broadly speaking). The repository is structured as a single package that is goverened by [`poetry`]. 
 
-### 🏠 [Homepage](https://stjude.cloud)
+You can use the snippet below to install the package and access the scripts outlined below.
 
-## Repository Structure
+```bash
+pip install stjudecloud-utilities
+```
 
-The repository is laid out as follows:
+## Scripts
 
-* `scripts` - A collection of utility scripts to be used with St. Jude Cloud genomics platform.
-  * `deduplicate.py` - A utility script to deduplicate St. Jude Cloud file names. Assumes files are annotated with St. Jude project names. Retains files based on the `PRIORITIES` list of prioritized projects.
-    * Usage: `dx ls /counts | python3 deduplicate.py | bash`
-  * `gtf_to_RSeQC_bed.py` - A utility script for converting Gencode GTFs into the custom BED12 format used by the [RSeQC software package][http://rseqc.sourceforge.net/].
-    * Usage: `python3 gtf_to_RSeQC_bed.py <gencode gtf> > gencode.vFOO.bed`
+The following scripts are included in the package:
 
-## Author
+### `deduplicate-samples`
```

Comment:
To me `deduplicate-samples` suggests that is the script name. I would probably either use the script name (or rename the script) or just rewrite the headers to describe the purpose, such as Deduplicate Feature Counts or something similar.

Reply:
If we rename, I would either use `deduplicate_feature_counts` or ensure that the script can deduplicate other file types (I think it can).